### PR TITLE
Windows: Remove meaningless warnings on docker info

### PR DESCRIPTION
**Pull request by lowenna**
Signed-off-by: John Howard jhoward@microsoft.com

Stops warnings from appearing when running docker info against a Windows daemon due to checking properties which are not supported on the platform.

### CODE REVIEW

**Comment by estesp** [comment]
seems reasonable.. LGTM

File: api/client/info.go [hunk 1]
```diff
@@ -70,20 +71,23 @@ func (cli *DockerCli) CmdInfo(args ...string) error {
...
-	if !info.IPv4Forwarding {
-		fmt.Fprintf(cli.err, "WARNING: IPv4 forwarding is disabled.\n")
-	}
-	if !info.BridgeNfIptables {
-		fmt.Fprintf(cli.err, "WARNING: bridge-nf-call-iptables is disabled\n")
-	}
-	if !info.BridgeNfIp6tables {
-		fmt.Fprintf(cli.err, "WARNING: bridge-nf-call-ip6tables is disabled\n")
+	// Only output these warnings on platforms where there is support.
+	if runtime.GOOS != "windows" {
```
Comment (duglin) [inline api/client/info.go]: Should it checking the runtime of the daemon or the cli?

**Comment by cpuguy83** [comment]
These warnings shouldn't be handled on the client, and should probably be on the info struct as `Warnings []string` which the client can just iterate through.

But basically yeah, I agree with @estesp this isn't the right change since the Windows client may very well be talking to a linux daemon.

**Comment by jhowardmsft** [comment]
Yeah, that PR for the server to return it's OS was one of mine :) So what was a couple of line change has turned into touching 19 files. Now it's fixed correctly I believe. Thanks for catching my stupidity on the first fix. :+1:

**Comment by jhowardmsft** [comment]
Ugh. Experimental invokes cli.call which I've had to change the return parameters to. As the call it makes is into libnetwork, I've had to add a wrapper to avoid changing libnetwork and revendoring. I can't think of a more elegant way around this without being more invasive. I can follow up with a subsequent PR to remove the wrapper after I submit a change to libnetwork/revendor and then update docker/docker.

**Comment by jhowardmsft** [comment]
21 files now :)

**Comment by duglin** [comment]
LGTM

**Comment by jhowardmsft** [comment]
(Mirroring conversation on IRC). @cpuguy83. There's pro's and cons of doing it either way. Both are equally valid. My rationale for keeping it in the client is purely to avoid a change to the API. But I'm happy to do either. [Regardless, the rework of using the serverResponse struct is surely for the better IMO]

**Comment by jfrazelle** [comment]
LGTM

**Pull request merged**